src/api/app/controllers/concerns/rescue_handler.rb

```diff
@@ -31,6 +31,10 @@ module RescueHandler
       render plain: text, status: http_status
     end
 
+    rescue_from Trigger::Errors::InvalidToken do |exception|
+      render_error status: 403, errcode: 'invalid_token', message: exception.message
```

Comment:
There's a typo:

```suggestion
      render_error status: 403, errorcode: 'invalid_token', message: exception.message
```

What is the error message users will see? Is it generic?

Reply:
if you look the [code](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/trigger/errors.rb#L7), we have there "No valid token found"